_posts/2018-07-25-strimzi-0-5-0-release.md

```diff
@@ -0,0 +1,133 @@
+```
+
+So `kafka` is now at same level as any other Kubernetes/OpenShift native resourse, for example `configmap`, `deployment`, `secret` and so on.
+The same is true for Kafka Connect cluster which is defined by a `KafkaConnect` Custom Resource.
+
+# Encryption support
+
+The communication across the deployed Kafka cluster is now encrypted using the TLS protocol and providing mutual authentication as well.
+More precisely, the following traffic that is now encrypted:
+
+* Zookeeper cluster communication
+* Kafka cluster commbunication
+* Communication between Kafka and Zookeeper
+* Communication between Topic Operator and Kafka / Zookeeper
+
+Setting up the TLS stuff is done by the Cluster Operator which generates the certificates needed for encryption.
+At same time a clients CA certificate is provided within a Secret so that the user can use it for signing clients certificates and allowing client authentication for those clients which wants to connect via TLS.
+
+# Logging configuration improvements
+
+In order to improve the logging experience, it's now possible to specify the logging level (i.e. `DEBUG`, `INFO`, ...) in the Custom Resource describing the cluster both for Kafka and Zookeeper.
+There are two different ways for doing this.
+
+The first one is about specifying the new log level "inline" in the cluster definition.
+
+```yaml
+logging:
+    type: inline
+    loggers:
+      logger.name: "INFO"
+```
+
+The other way is much more powerful because it's possible to provide the full log4j configuration through a `log4j.properties` file inside a ConfigMap and referring to it from the cluster description.
+
+```yaml
+logging:
+    type: external
+    name: customConfigMap
+```
+
+# Rack awareness
+
+In order to have high availability, Apache Kafka provides an interesting feature called "rack awareness".
+When your topics are replicated across multiple brokers, it's really useful that these brokers live in different physical racks.
+If a rack fails, you are sure that other topic replicas are alive, a new leader can be elected and clients can continue to exchange messages. As opposite, if the brokers hosting replicas for a topic are put into the same rack, the partitions are not available anymore on rack failure because all brokers will go down at same time.
+
+The new Strimzi rack awareness feature provides a way to specify a `topologyKey` in order to spread brokers across different nodes.
+The `topologyKey` defines a node label which is used as `broker.rack` for configuring the Kafka brokers accordingly.
+
+```yaml
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+    rack:
+      topologyKey: failure-domain.beta.kubernetes.io/zone
+    # ...
+```
+
+When deploying the Kafka cluster in the public Cloud (i.e. Amazon, Azure and GCP), it means spreading brokers across different availability zones which can be compared to the physical racks.
+
+# Affinity, Taints and Tolerations
```

Comment:
We actually do not support Taints. That is something related to nodes only. So This section should IMHO list only Affinity and Toleration